woocommerce/woocommerce
# Update WC blocks e2e tests to WordPress 6.6

### Changes proposed in this Pull Request:

This PR updates WC blocks e2e tests to work with WordPress 6.6. Most updates were related to changes in the Site Editor UI.

### How to test the changes in this Pull Request:

_(No need to test when testing the release)_

1. Make sure e2e tests pass.
2. Make sure code changes make sense.

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
